### ISSUE TITLE
Is possible to connect with Nicehash stratum servers?

### ISSUE THREAD

**Issue by mazetuski** [cazala/coin-hive-stratum#127]
It always says that it is an invalid address

![imagen](https://user-images.githubusercontent.com/11924094/34294711-135fad00-e70a-11e7-99fd-ca2c44783e04.png)

**Comment by matthewlilley** [comment]
Not sure that this has support for NiceHash,  maybe wrong but the last time I tried it wasn't supported.

**Comment by Baaleos** [comment]
I dont think that is the correct connection string - try it with just the hostname and port
eg: minus the tcp+stratum://

**Comment by akramtariqkhan** [comment]
Any solution to make it work with nicehash ?

**Comment by abale** [comment]
I think the issue is mostly on the client's code side - it needs to allow non-wallet addresses in the pool username.

**Comment by d668** [comment]
hey, any update on this? How can I make coin-hive-stratum talk to Nicehash stratum servers? what am I supposed to put in /config/defaults.js ?
